lua/hawtkeys/ts.lua

```diff
@@ -341,24 +341,43 @@ local function find_maps_in_file(filePath)
                     )
                     break
                 end
+                local unknown_value = { "unknown" }
                 local strObj =
                     vim.treesitter.get_node_text(node.node, fileContent)
                 local ok, tableObj = pcall(function()
+                    local env = setmetatable({}, {
+                        __index = function()
+                            setmetatable(unknown_value, {
+                                __index = function()
+                                    return unknown_value
+                                end,
+                            })
+                            return unknown_value
+                        end,
+                    })
                     --Remove wrapping parens and wrap in table and unpack - issue #81
                     strObj = strObj:gsub("^%s*%(%s*", ""):gsub("%s*%)%s*$", "")
-                    return loadstring("return {" .. strObj .. "}")()
+                    local func = loadstring("return {" .. strObj .. "}")
```

Comment:
I think you may need upvalues as well, and this could still throw an error in some cases I think, like when the value returned here is not what's expected.